oci/casext/refname.go

```diff
@@ -76,8 +69,7 @@ func (e Engine) ResolveReference(ctx context.Context, refname string) ([]Descrip
 	// the time of writing -- 1.0.0-rc5) there are some discussions to add this
 	// restriction in 1.0.0-rc6.
 	for _, descriptor := range index.Manifests {
-		// XXX: What should we do if refname == "".
-		if descriptor.Annotations[ispec.AnnotationRefName] == refname {
+		if descriptor.Digest.String() == refname {
```

Comment:
This change is also incorrect -- it breaks basically all of the `umoci` command-line tools because now reference resolution requires you to specify the digest (this code is what actually figures out what `ubuntu:latest` means in an OCI image).